### PR TITLE
Added osx_arm64

### PR DESCRIPTION
**Pull request by pariterre**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pariterre** [comment]
@conda-forge-admin, please rerender

**Comment by pariterre** [comment]
@conda-forge-admin, please rerender

**Comment by pariterre** [comment]
@conda-forge-admin, please rerender

**Comment by pariterre** [comment]
@conda-forge/help-c-cpp @conda-forge/help-python 
Hello! 
I am having some problems adding the osx_arm64. For some reason, it cannot find the Numpy, while it seems to find Python (but the wrong one). Can anyone help? :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,15 +23,18 @@ requirements:
...
     - pkgconfig
-    - {{ compiler('cxx') }}
-  host:
-    - rbdl >=3.1.3
-    - ipopt
+    - {{ pin_compatible('rbdl') }}
     - tinyxml
-    - numpy
+    - ipopt
     - python
```
Comment (dopplershift) [inline recipe/meta.yaml:29]:
```suggestion
```

**Comment by dopplershift** [comment]
Ok, I was trying to do this from memory, but it's easier to use something that works:

https://github.com/conda-forge/matplotlib-feedstock/blob/637c2286194479d74e97310b23930ea1ae1fc4f4/recipe/meta.yaml#L25-L60

**Comment by pariterre** [comment]
@conda-forge-admin, please rerender

**Pull request merged**